cmd/bridge/menu.go

```
@@ -0,0 +1,425 @@
+	return -1
+}
+
+// actOpenAdmin opens the local admin console URL in the user's
+// browser. Best-effort — on a headless host the open command will
+// fail and we just print the URL instead.
+func actOpenAdmin(_ context.Context, _ *bufio.Reader, stdout, _ io.Writer, _ menuState) int {
+	url := "http://127.0.0.1:7789/"
+	fmt.Fprintf(stdout, "  Admin console: %s\n", url)
+	switch runtime.GOOS {
+	case "darwin":
+		_ = exec.Command("open", url).Start()
```

golangci-lint (2.11.4)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)